test/e2e_closure_test.ts

```diff
@@ -58,7 +59,7 @@ describe('golden file tests', () => {
       'warning_level': 'VERBOSE',
       'js': goldenJs,
       'externs': externs,
-      'language_in': 'ECMASCRIPT6_STRICT',
+      'language_in': 'STABLE',
```

Comment:
why this change?

Reply:
Better matches the recommended configuration, but it's an unrelated change, better put into its own PR